src/main/java/jenkins/branch/MultiBranchProjectFactory.java

```diff
@@ -205,7 +205,7 @@ public MultiBranchProjectFactoryDescriptor getDescriptor() {
          * @param source a repository
          * @return criteria for treating its branches as a match
          */
-        @NonNull
+        @CheckForNull
```

Comment:
How on earth is this related? And the Javadoc does not describe what a null return value would mean.